src/jit/flowgraph.cpp

```diff
@@ -24931,3 +24934,26 @@ unsigned Compiler::fgMeasureIR()
 
     return nodeCount;
 }
+
+//------------------------------------------------------------------------
+// fgCompDominatedByExceptionalEntryBlocks: compute blocks that are
+// dominated by not normal entry.
+//
+void Compiler::fgCompDominatedByExceptionalEntryBlocks()
+{
+    for (unsigned i = 1; i <= fgBBNumMax; ++i)
+    {
```

Comment:
You don't need to run this loop at all if fgEnterBlksSet contains exactly one element, since that one element will be the normal entry block (you could assert this).